Getting error due to deprecated dependency in this package- ##[error] WARN deprecated lightcookie@1.0.25: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.

I am getting error on sonar vaidation - ##[error] WARN deprecated lightcookie@1.0.25: Package no longer supported. Contact Support at https://www.npmjs.com/support for more info.
This lightcookie package is a dependency in pactum package
Version- 3.7.5 and 3.7.6

<img width="1158" alt="image" src="https://github.com/user-attachments/assets/fbda90a8-45a6-44e3-92e7-c24cb34dbf2f" />

--- a/package.json
+++ b/package.json
@@ -58,7 +58,7 @@
     "form-data-lite": "^1.0.3",
     "json-query": "^2.2.2",
     "klona": "^2.0.6",
-    "lightcookie": "^1.0.25",
+    "cookie-lite": "^0.0.1",
     "openapi-fuzzer-core": "^1.0.6",
     "pactum-matchers": "^1.1.7",
     "parse-graphql": "^1.0.0",

--- a/src/helpers/toss.helper.js
+++ b/src/helpers/toss.helper.js
@@ -1,4 +1,4 @@
-const lc = require('lightcookie');
+const cl = require('cookie-lite');
 const jqy = require('json-query');
 
 const config = require('../config');
@@ -30,7 +30,7 @@ function getPathValueFromRequestResponse(path, request, response) {
     if (!request.headers) {
       request.headers = {};
     }
-    const cookies = lc.parse(request.headers['cookie']);
+    const cookies = cl.parse(request.headers['cookie']);
     data = cookies;
   } else if (path.startsWith('req.body')) {
     path = path.replace('req.body', '');
@@ -43,7 +43,7 @@ function getPathValueFromRequestResponse(path, request, response) {
     if (!response.headers) {
       response.headers = {};
     }
-    const cookies = lc.parse(response.headers['set-cookie']);
+    const cookies = cl.parse(response.headers['set-cookie']);
     data = cookies;
   } else {
     path = path.replace('res.body', '');

--- a/src/helpers/utils.js
+++ b/src/helpers/utils.js
@@ -1,6 +1,6 @@
 const { compare } = require('pactum-matchers').utils;
 const graphQL = require('./graphQL');
-const lc = require('lightcookie');
+const cl = require('cookie-lite');
 const { PactumRequestError } = require('../helpers/errors');
 
 const log = require('../plugins/logger');
@@ -69,7 +69,7 @@ const utils = {
       if (value !== undefined) {
         cookieObject[key] = value;
       } else if (value === undefined) {
-        cookieObject = lc.parse(key);
+        cookieObject = cl.parse(key);
       }
     } else {
       if (!helper.isValidObject(key)) {

--- a/src/models/Interaction.model.js
+++ b/src/models/Interaction.model.js
@@ -1,4 +1,4 @@
-const lc = require('lightcookie');
+const cl = require('cookie-lite');
 const { setMatchingRules, getValue } = require('pactum-matchers').utils;
 const processor = require('../helpers/dataProcessor');
 const helper = require('../helpers/helper');
@@ -121,7 +121,7 @@ class InteractionRequest {
       this.headers = getValue(request.headers);
     }
     if (request.cookies && typeof request.cookies === 'object') {
-      const cookie = lc.serialize(request.cookies);
+      const cookie = cl.serialize(request.cookies);
       if (!this.headers) {
         this.headers = {};
       }
@@ -170,7 +170,7 @@ class InteractionResponse {
     setMatchingRules(this.matchingRules, response.body, '$.body');
     this.body = getValue(response.body);
     if (response.cookies) {
-      const cookie = lc.serialize(response.cookies);
+      const cookie = cl.serialize(response.cookies);
       if (!this.headers) {
         this.headers = {};
       }

--- a/src/models/Spec.js
+++ b/src/models/Spec.js
@@ -1,5 +1,5 @@
 const fs = require('fs');
-const lc = require('lightcookie');
+const cl = require('cookie-lite');
 const override = require('deep-override');
 const path = require('path');
 const Tosser = require('./Tosser');
@@ -244,7 +244,7 @@ class Spec {
     }
     let cookie;
     if (typeof key === 'object') {
-      cookie = lc.serialize(key);
+      cookie = cl.serialize(key);
     } else {
       if (value) {
         cookie = `${key}=${value}`;

--- a/src/models/expect.js
+++ b/src/models/expect.js
@@ -1,6 +1,6 @@
 const assert = require('assert');
 const jqy = require('json-query');
-const lc = require('lightcookie');
+const cl = require('cookie-lite');
 
 const config = require('../config');
 const utils = require('../helpers/utils');
@@ -115,7 +115,7 @@ class Expect {
       if (!actualCookie) {
         this.fail(`'set-cookie' key not found in response headers`);
       }
-      actualCookie = lc.parse(actualCookie);
+      actualCookie = cl.parse(actualCookie);
       assert.deepStrictEqual(actualCookie, expectedCookie);
     }
   }
@@ -131,7 +131,7 @@ class Expect {
       if (Array.isArray(actualCookie) && actualCookie.length > 1) {
         actualCookie = actualCookie.join('; ') + ';';
       }
-      actualCookie = lc.parse(actualCookie);
+      actualCookie = cl.parse(actualCookie);
       const msg = jlv.validate(actualCookie, expectedCookie, { target: 'Cookie' });
       if (msg) this.fail(msg);
     }

